### PR TITLE
Docs: Complete `checklists` for `GSoC`

### PR DESCRIPTION
**Pull request by Abinashbunty**
- Added my name to the list of participants
- Completed the checklists which I could complete until now.
- More updates about writing blog and Django/Wagtail will be completed soon! 😄

### CODE REVIEW

**Comment by Abinashbunty** [comment]
@thibaudcolas I'm quite interested in the `Greener Coding` project. Looking forward to collaborate and learn in the process. 🙂

**Comment by KingNonso** [comment]
Looks clean and superb 👍  @Abinashbunty 
That writing part... here something to get you cracking from the last Wagtail webinar. You can now watch it back here: https://youtu.be/f0d4hW2nzmQ
- UX enhancements feedback: https://github.com/wagtail/wagtail/discussions/9553 
- Outreachy: https://wagtail.org/blog/our-very-first-outreachy-interns/ 
- Accessibility checker: https://wagtail.org/blog/introducing-wagtails-new-accessibility-checker/ 
- Editor guide: https://guide.wagtail.org/ 
- Snippets vs ModelAdmin parity: https://github.com/wagtail/wagtail/discussions/10206

File: 2023/Abinash Satapathy/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+| Twitter          |  <a href="https://twitter.com/abinashbunty" target="blank"><img align="center" src="https://raw.githubusercontent.com/Abinashbunty/icons/c2a12af3046730ea2d0751aa59c4f1503951a56f/5296514_bird_tweet_twitter_twitter%20logo_icon.svg" alt="abinashbunty" height="20" width="20" /></a>    |
+| Mastodon         |   <a href="https://me.dm/@Abinashbunty" target="blank"><img align="center" src="https://raw.githubusercontent.com/Abinashbunty/icons/c2a12af3046730ea2d0751aa59c4f1503951a56f/4518932_mastodon_icon.svg" alt="abinashbunty" height="20" width="20" /></a>   |
+| LinkedIn         |   <a href="https://www.linkedin.com/in/abinashbunty/" target="blank"><img align="center" src="https://raw.githubusercontent.com/Abinashbunty/icons/c2a12af3046730ea2d0751aa59c4f1503951a56f/5296501_linkedin_network_linkedin%20logo_icon.svg" alt="abinashbunty" height="20" width="20" /></a>   |
+| Personal website |   <a href="https://www.abinash.nl/" target="blank"><img align="center" src="https://raw.githubusercontent.com/Abinashbunty/icons/c2a12af3046730ea2d0751aa59c4f1503951a56f/984119_avatar_male_man_person_user_icon.svg" alt="abinashbunty" height="20" width="20" /></a>   |
```
Comment (thibaudcolas) [inline 2023/Abinash Satapathy/README.md:18]: Cool stuff!

File: 2023/Abinash Satapathy/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Here, we expect you to go through two tutorials to demonstrate your understanding of Django and Wagtail. Submit both projects you will have created as a pull request on this `gsoc` repository.
+
+- [ ] Go through Django’s [Getting started documentation and tutorial](https://docs.djangoproject.com/en/4.1/intro/), until part 7.
+- [ ] Go through Wagtail’s Getting started tutorial: [Your first Wagtail site](https://docs.wagtail.org/en/stable/getting_started/tutorial.html)
+
+
+## Research
+
+- I'm interested to contribute to the `Greener coding` project. Often while developing applications, I tend to focus on the fast response time of the application. But I have never thought much about the energy consumption of the application. I'm interested to learn more about this project and contribute to it.
+- [Create Low-Carbon Images](https://climateaction.tech/actions/create-low-carbon-images/) - This provides some information about optimizing images for the web. I'm interested to learn more about this.
```
Comment (thibaudcolas) [inline 2023/Abinash Satapathy/README.md:65]: That’s a great resource.

File: contributor-guide.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ This checklist helps you demonstrate your understanding of how people use GitHub
 - [ ] Now make another pull request to your own README file, adding a Markdown table with links to:
   - [ ] Your GitHub profile
   - [ ] Your Twitter profile if any
-  - [ ] Your Mastodown profile if any
+  - [ ] Your Mastodon profile if any
```
Comment (thibaudcolas) [inline contributor-guide.md:62]: 😄 😅

**Review by thibaudcolas — APPROVED** [review]
Thank you for taking the time to describe your pull request @Abinashbunty! Very few people have done this so far. Also appreciate you taking the time to fix those typos.

You seem to have quite a bit of experience of open source – are you sure GSoC is still for you? I forget what Google’s specific eligibility criteria are, worth double checking.

I think your experience / interest in serverless will be very relevant to the green coding project. It’s a good avenue for improvement for us if a Wagtail site could "sleep" while not in use, as would be the case with a "scale to zero" serverless model.

**Comment by Abinashbunty** [comment]
> Thank you for taking the time to describe your pull request @Abinashbunty! Very few people have done this so far. Also appreciate you taking the time to fix those typos.
> 
> You seem to have quite a bit of experience of open source – are you sure GSoC is still for you? I forget what Google’s specific eligibility criteria are, worth double checking.
> 
> I think your experience / interest in serverless will be very relevant to the green coding project. It’s a good avenue for improvement for us if a Wagtail site could "sleep" while not in use, as would be the case with a "scale to zero" serverless model.

@thibaudcolas Although I've been part of open-source community since the past Hacktoberfest, I haven't been involved in a project in a major way. That restricts me from learning more. Also I had this dream to be part of GSoC before I finish my studies. 😄 And @wagtail seemed to be the perfect fit for me at this point in career.

File: contributor-guide.md (hunk 3 above)
Comment (Abinashbunty) [reply contributor-guide.md:62]: Haha, it happens in a hurry. 😉

**Comment by Abinashbunty** [comment]
And yes, I am eligible for GSoC which is why I'm applying in fact. 😅

**Comment by thibaudcolas** [comment]
🤘 if you’re happy, I’m happy

**Pull request merged**